src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/JsonPropertyInfo.cs

```diff
@@ -172,6 +172,30 @@ internal JsonConverter EffectiveConverter
             }
         }
 
+        private JsonObjectCreationHandling? _objectCreationHandling;
+        internal JsonObjectCreationHandling EffectiveObjectCreationHandling { get; private set; }
+
+        /// <summary>
+        /// Gets of sets a value indicating if member will be replaced or populated during deserialization.
+        /// When <see langword="null"/> default value will be used.
+        /// </summary>
+        /// <remarks>
+        /// Default value is resolved based on capability of property converter to populate,
```

Comment:
When you say "default value" here, do you mean the effective handling computed on the configure stage?

Reply:
Will be fixed in latest iteration. New comment:

```csharp
        /// <summary>
        /// Gets or sets a value indicating if the property or field should be replaced or populated during deserialization.
        /// </summary>
        /// <remarks>
        /// Initial value for this property is based on the presence of <see cref="JsonObjectCreationHandlingAttribute"/> attribute on the property.
        /// When <see langword="null"/> effective handling will be resolved based on
        /// capability of property converter to populate, containing type's <see cref="JsonTypeInfo.PreferredPropertyObjectCreationHandling"/>.
        /// and <see cref="JsonSerializerOptions.PreferredObjectCreationHandling"/> value.
        /// </remarks>
```